projects/canopy/src/lib/pagination/pagination.component.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../styles/mixins';
+@import '../../styles/variables/components/pagination';
```

Comment:
No need to add this import here. Instead, could you please add it to the `variables.scss` please?

Reply:
removed